lib/index.js

```diff
@@ -159,7 +175,14 @@ function getPolyfillString(options) {
 					}
 				});
 				if (builtPolyfillString) {
-
+					return streamToString(
+						browserify(intoStream(builtPolyfillString))
```

Comment:
Agree, we should only browserify if at least one polyfill has `build.commonjs` = true